perfkitbenchmarker/benchmark_spec.py

```diff
@@ -228,6 +233,32 @@ def ConstructVirtualMachines(self):
       self.vm_groups[group_name] = vms
       self.vms.extend(vms)
 
+  def ConstructSparkService(self):
+    if self.config.spark_service:
```

Comment:
Why not
```
if self.config.spark_service is None:
  return
```
like you did above for `ConstructVirtualMachines`? Fewer indent levels makes it easier for me to read things.